modules/nw-enforcing-hsts-per-domain.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+// * networking/configuring-routing.adoc
+
+[id="nw-enforcing-hsts-per-domain_{context}"]
+= Enforcing HTTP strict transport security per-domain
+
+To enforce HTTP strict transport security (HSTS) per-domain for all TLS routes, add a `requiredHSTSPolicies` record to the Ingress spec to capture the configuration of the HSTS policy.
+
+If you configure the `route.openshift.io/RequiredRouteAnnotations` plug-in to enforce HSTS, then any newly created route must be configured with a compliant HSTS policy, which is verified against the global setting on the cluster Ingress configuration, called `ingresses.config.openshift.io/cluster`.
+
+[NOTE]
+====
+To handle upgraded clusters with non-compliant HSTS routes, you can update the manifests at the source and apply the updates.
+====
+
+[IMPORTANT]
+====
+HSTS cannot be applied to non-TLS routes, even if HSTS is requested for all routes globally.
```

Comment:
Sometimes we say secure, other times we say TLS, probably using secure instead of TLS is better.  Or just pick the one that corresponds with other docs.